packages/dashboard/src/components/common/Button.tsx

```diff
@@ -1,25 +1,108 @@
-import type { MouseEventHandler } from "react";
+import React, { ReactNode } from "react";
+import { classNames } from "src/functions";
 
-interface Props {
-  disabled?: boolean;
-  onClick: MouseEventHandler<HTMLElement>;
-  text: string;
-}
+export type ButtonColor = "blue" | "red" | "brown" | "default";
+export type ButtonSize = "xs" | "sm" | "lg";
+export type ButtonVariant = "outlined" | "filled" | "empty";
+
+const DIMENSIONS = {
```

Comment:
Should these style-related aspects be in a separate file? I don't feel super strongly about it but that might be nicer.

Reply:
Do they relate to anything outside this file? If they're only related to `<Button />` then they should probably live in this file

Reply:
its all button centric config. sorry I wasn't getting notices for this PR.